File: testprojects/python_main/subgraphs/all_type/code/org/python/all_type_main_1/RowDistributor_1_1_1.py
from pyspark.sql import *
from pyspark.sql.functions import *
from pyspark.sql.types import *
from python_streaming_main.config.ConfigStore import *
from python_streaming_main.udfs.UDFs import *

def RowDistributor_1_1_1(spark: SparkSession, in0: DataFrame) -> (DataFrame, DataFrame):
    df1 = in0.filter((col("`c- short`") > lit(-1)))
    df2 = in0.filter((col("`c- short`") > lit(5)))

    return df1, df2

File: testprojects/scala_main/pipelines/PERF_SC_PIP/code/src/main/scala/org/main/scla_dep_mgmt_change/graph/PERF_SG/RowDistributor_1_1_1.scala
package org.main.scla_dep_mgmt_change.graph.PERF_SG

import io.prophecy.libs._
import org.main.scla_dep_mgmt_change.config.ConfigStore._
import org.main.scla_dep_mgmt_change.config.Context
import org.main.scla_dep_mgmt_change.udfs.UDFs._
import org.main.scla_dep_mgmt_change.udfs._
import org.apache.spark._
import org.apache.spark.sql._
import org.apache.spark.sql.functions._
import org.apache.spark.sql.types._
import org.apache.spark.sql.expressions._
import java.time._

object RowDistributor_1_1_1 {

  def apply(context: Context, in: DataFrame): (DataFrame, DataFrame) =
    (in.filter(!col("cmls_3ds_authntn_mthd").like("%mister transistor%")),
     in.filter(col("cmls_acct_cobrnd_bus_id_drvd") > lit(-100))
    )

}
